dev apps/DesktopTestApp/MainForm.cs

```diff
@@ -105,33 +104,34 @@ private void confidentialClient_Click(object sender, EventArgs e)
                 IAuthenticationResult result;
                 if (userList.SelectedIndex != -1)
                 {
-                    if (modalWebview.Checked)
-                    {
+                    //if (modalWebview.Checked)
```

Comment:
The modalWebView is non-existant at this point, but I wanted to leave the code in for future work, which is why I commented it out